src/lib/serviceRoutinesV2/getEntities.cpp

```diff
@@ -82,6 +83,7 @@ std::string getEntities
   std::string  id         = ciP->uriParam["id"];
   std::string  idPattern  = ciP->uriParam["idPattern"];
   std::string  q          = ciP->uriParam["q"];
+  std::string  mq         = ciP->uriParam["mq"];
```

Comment:
Again, if "mq" refers to the name of an URI param (it seems so by `uriParam` :)) the URI_PARAM_MQ should be used. Is that correct?